README.md

```diff
@@ -169,8 +176,6 @@ If you like this project it would be truly appreciated if you give it a ⭐️.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./MIT) licensed.
```

Comment:
- [ ] Please fix the MIT license link as requested by the previous reviewer; now, when I press it, I get a 404 page.
### Hint: try to change the path to `./MIT.md` instead of `./MIT`